pkg/templateservicebroker/util/util.go

```diff
@@ -0,0 +1,48 @@
+package util
```

Comment:
i'm not convinced this is the right location for this helper.  it doesn't seem TSB specific.  Perhaps it even belongs under pkg/authorization/util/external or something, which would put it closer to where it came from?

@deads2k wdyt?

Reply:
> i'm not convinced this is the right location for this helper. it doesn't seem TSB specific. Perhaps it even belongs under pkg/authorization/util/external or something, which would put it closer to where it came from?
> 
> @deads2k wdyt?

After we can get a 3.8 rebase and make openshift/client-go authoritative, we could compare it for similar code and see if we have something generic to add.  Until then, I'd suggest trying to limit TSB's exposure to openshift/origin by whatever means necessary